Resources/config/doctrine/JobLog.orm.yml

```diff
@@ -0,0 +1,37 @@
+Markup\JobQueueBundle\Entity\JobLog:
+  type: entity
+  table: job_log
+  id:
+    uuid:
+      type: string
+      unique: true
+      length: 100
+  fields:
+    command:
+      type: string
+      length: 2048
+    topic:
+      type: string
+      length: 60
+    added:
+      type: datetime
+    started:
+      type: datetime
+      nullable: true
+    completed:
+      type: datetime
+      nullable: true
+    status:
+      type: string
+      length: 60
+    output:
+      type: string
+      length: 2048
+    peakMemoryUse:
+      type: integer
+      length: 255
+      nullable: true
+    exitCode:
+      type: string
+      length: 60
+      nullable: true
```

Comment:
ExitCode likely shouldnt be nullable, very command has an exit code.

Reply:
it's an exit code of exception that we store here